### PR TITLE
Add masking support to BatchNormalization layer

### PR DESCRIPTION
**Pull request by alshedivat**
A one-line fix for issue #1962.

### CODE REVIEW

File: keras/layers/normalization.py [hunk 1]
```diff
@@ -52,11 +52,16 @@ class BatchNormalization(Layer):
     # Output shape
         Same shape as input.
 
+    # Masking
```
Comment (fchollet) [inline keras/layers/normalization.py]: This comment is not necessary.

Comment (alshedivat) [inline keras/layers/normalization.py]: Removed.

**Comment by fchollet** [comment]
LGTM

**Comment by ttrine** [comment]
Hi there, thanks for the fix.

However are we sure it is correct? After some investigation it looks to me like the 0s in 0-padded sequences distort the batch statistics.

I'd appreciate if someone could comment.
